### ISSUE TITLE
p:selectBooleanCheckbox / p:selectOneMenu lack keyboard focus CSS styling

### ISSUE THREAD

**Issue by codebulb** [codebulb/crudfaces#21]
For both `<p:selectBooleanCheckbox>` and `<p:selectOneMenu>`, keyboard focus should be represented by a "blue glowing border", as is the case for `<p:inputText>`.

**Comment by codebulb** [comment]
Fixed.
